tools/Evm/T8n/JsonConverters/AccountStateJsonConverter.cs

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Ethereum.Test.Base;
+using Nethermind.Core.Extensions;
+using Nethermind.Int256;
+using Nethermind.Serialization.Json;
+
+namespace Evm.T8n.JsonConverters;
+
+// required to serialize in geth t8n format
+public class AccountStateJsonConverter : JsonConverter<AccountState>
```

Comment:
Do we really need it? Wouldn't just setting some attributes on `AccountState` suffice?
I think we should move to serializing numbers same as Geth! @MarekM25 @smartprogrammer93 ?

Reply:
I agree